packages/desktop/src/rtl-tests/generalChannel.create.test.tsx

```diff
@@ -1,105 +0,0 @@
-import React from 'react'
-import { act } from 'react-dom/test-utils'
-import '@testing-library/jest-dom/extend-expect'
-import { apply, fork, take } from 'typed-redux-saga'
-import { renderComponent } from '../renderer/testUtils/renderComponent'
-import { prepareStore } from '../renderer/testUtils/prepareStore'
-import MockedSocket from 'socket.io-mock'
-import { ioMock } from '../shared/setupTests'
-import { socketEventData } from '../renderer/testUtils/socket'
-import { AnyAction } from 'redux'
-import { identity, publicChannels, getFactory, ChannelsReplicatedPayload } from '@quiet/state-manager'
-import { SocketActionTypes } from '@quiet/types'
-import Channel from '../renderer/components/Channel/Channel'
-
-jest.setTimeout(20_000)
-
-describe('General channel', () => {
```

Comment:
Why is it deleted?

Reply:
I think this is fully tested in community.create and community.join tests